android-json-form-wizard/src/main/java/com/vijay/jsonwizard/task/AttachRepeatingGroupTask.java

```diff
@@ -72,6 +74,7 @@ public AttachRepeatingGroupTask(final ViewParent parent, int numRepeatingGroups,
     @Override
     protected void onPreExecute() {
         showProgressDialog(R.string.please_wait_title, R.string.creating_repeating_group_message, widgetArgs.getFormFragment().getContext());
+        repeatingGroups = new ArrayList<>();
```

Comment:
What is this arrayList for?

Reply:
Thanks for catching that, I have removed